lib/iris/cube.py

```diff
@@ -1398,6 +1398,15 @@ def shape(self):
         return shape
 
     @property
+    def dtype(self):
+        """
+        Return the data type of this cube without causing the data payload
```

Comment:
1. Because this is a property it needs to be documented as if it were a simple attribute. Instead of "Return the data type ..." it should just be "The data type ...".
2. The docstrings for the `shape` and `ndim` properties refer to "the data of this cube", as opposed to just "this cube". It would be nice to keep them all consistent.
3. Perhaps move the comment about loading data into a second sentence.